src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.xml

```diff
@@ -10,12 +10,6 @@
         <service id="data_collector.config" class="Symfony\Component\HttpKernel\DataCollector\ConfigDataCollector">
             <tag name="data_collector" template="@WebProfiler/Collector/config.html.twig" id="config" priority="-255" />
             <call method="setKernel"><argument type="service" id="kernel" on-invalid="ignore" /></call>
-            <argument type="service">
-                <service class="string">
-                    <factory function="basename" />
-                    <argument>%kernel.project_dir%</argument>
```

Comment:
I added this less than one week ago.  thought it would be useful to display this when switching apps. Sure it's not?

Reply:
ah :D i guess not? the side effect is now `getApplicationName()` triggers, whereas before only `getAppName()`. Which are totally different things, obviously.

IMHO we should tailor to SF by default, any custom branding etc. can be done by collectors. This is about the Symfony config.

Reply:
This shows also in the toolbar, which is where it's potentially a useful hint. But ok to remove: the app's screens should already be a good enough hint.

Reply:
but not useful enough (see first screenshot)